src/test/kotlin/com/terraformation/backend/tracking/db/ObservationStoreTest.kt

```diff
@@ -1871,4 +1874,169 @@ class ObservationStoreTest : DatabaseTest(), RunsAsUser {
+                          )))),
+          ObservationZone(
+              zoneId = zoneId2,
+              plots =
+                  listOf(
+                      ObservationPlot(
+                          zone2PlotId1,
+                          listOf(
+                              PlantTotals(speciesId1, live = 1),
+                              PlantTotals(speciesId3, live = 1),
+                          )))))
+
+      val observationId2 = insertObservation(completedTime = Instant.ofEpochSecond(2))
+
+      helper.insertObservationScenario(
+          ObservationZone(
+              zoneId = zoneId1,
+              plots =
+                  listOf(
+                      ObservationPlot(
+                          zone1PlotId1,
+                          listOf(
+                              PlantTotals(speciesId1, live = 1, dead = 1, existing = 1),
+                              PlantTotals(speciesId2, live = 1),
+                              PlantTotals("fern", live = 2),
+                          ),
+                      ),
+                      ObservationPlot(
+                          zone1PlotId2,
+                          listOf(
+                              PlantTotals(speciesId2, live = 1),
+                          ))),
+          ))
+
+      store.removePlotFromTotals(zone1PlotId1)
+
+      helper.assertTotals(
+          setOf(
+              ObservedPlotSpeciesTotalsRow(
+                  observationId = observationId1,
+                  monitoringPlotId = zone1PlotId1,
+                  speciesId = speciesId1,
+                  speciesName = null,
+                  certaintyId = Known,
+                  totalLive = 2,
+                  totalDead = 1,
+                  totalExisting = 1,
+                  mortalityRate = 33,
+                  cumulativeDead = 1,
+                  permanentLive = 2),
+              // Parameter names omitted after this to keep the test method size manageable.
+              ObservedPlotSpeciesTotalsRow(
+                  observationId1, zone1PlotId1, speciesId2, null, Known, 1, 0, 0, 0, 0, 1),
+              ObservedPlotSpeciesTotalsRow(
+                  observationId1, zone1PlotId1, null, "fern", Other, 1, 1, 0, 50, 1, 1),
+              ObservedPlotSpeciesTotalsRow(
+                  observationId1, zone1PlotId2, speciesId2, null, Known, 1, 1, 0, 50, 1, 1),
+              ObservedPlotSpeciesTotalsRow(
+                  observationId1, zone2PlotId1, speciesId1, null, Known, 1, 0, 0, 0, 0, 1),
+              ObservedPlotSpeciesTotalsRow(
+                  observationId1, zone2PlotId1, speciesId3, null, Known, 1, 0, 0, 0, 0, 1),
+              ObservedZoneSpeciesTotalsRow(
+                  observationId1, zoneId1, speciesId1, null, Known, 0, 0, 0, 0, 0, 0),
+              ObservedZoneSpeciesTotalsRow(
+                  observationId1, zoneId1, speciesId2, null, Known, 1, 1, 0, 50, 1, 1),
+              ObservedZoneSpeciesTotalsRow(
```

Comment:
I am just realizing that this may cause some confusion depending on how the FE is displaying observation level statistics.

My original intent when writing this section of the PRD was that the plant dashboard statistics should be updated to reflect the removal of the plots and their data, but observation level statistics themselves would be unchanged (even for zones/planting sites).

I realized this is not clear at all in the reqs but I will probably have to think through it a bit to see if we want to change this at all.

Reply:
I believe the dashboard shows the numbers from the most recent observation. There's no concept of observation-independent observation results in the system.

Reply:
Yeah that makes sense.  Okay well not worth changing anything at this point I think.  This will work fine and if it leads to confusion in the future we can solve with UI tweaks.

Reply:
The dashboard pulls information from a few different endpoints (since it also shows things like reported plants that aren't related to observations) but for the observation data, I believe it fetches the full list of all observation results and shows the data from the most recent one. I think it fetches the whole list so it doesn't have to re-fetch if you look at the history views, but one of the frontend devs may know better.

Reply:
My apologies for the misleading on that.